build.ps1

```diff
@@ -113,10 +113,9 @@ Invoke-BuildStep 'Cleaning artifacts' {
     -skip:($Fast -or $SkipXProj) `
     -ev +BuildErrors
 
-Invoke-BuildStep 'Enabling delay-signing' {
-        Enable-DelaySigning $MSPFXPath $NuGetPFXPath
+Invoke-BuildStep 'Configure delay signing' {
+        Configure-DelaySigning -MSPFXPath:$MSPFXPath -NuGetPFXPath:$NuGetPFXPath -CI:$CI
```

Comment:
Yes, needed to set a boolean value to a switch parameter explicitly. Not needed for other parameters.